src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -224,6 +224,17 @@ trait WorkflowComponent {
       findWorkflowsBySubmissionId(submissionId).filter(_.status inSet(statuses.map(_.toString))).result
     }
 
+    def countWorkflowsByQueueStatus: ReadAction[Map[String, Int]] = {
+      findQueuedAndRunningWorkflows.result.map { recs =>
+        val grouped = recs.groupBy {
+          // lump all of the workflows currently being handled by Cromwell into "Active"
+          case runningRecord if WorkflowStatuses.runningStatuses.map(_.toString).toSet contains runningRecord.status => "Active"
```

Comment:
why are we doing this lumping here? I think this should be a straight group by in the db and let the caller (ui?) figure out how they want to lump.